dev-docs/modules/optimeraRtdProvider.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: page_v2
+title: Optimera Real Time Data Module
+display_name: Optimera RTD
+description: Optimera Optimization Targeting
+page_type: module
+module_type: rtd
+module_code : optimeraRTD
```

Comment:
The module_code needs to match the file name. So this should be optimeraRtdProvider